app/controllers/admin/domain_blocks_controller.rb

```diff
@@ -33,7 +33,7 @@ def create
 
       # Disallow accidentally downgrading a domain block
       if existing_domain_block.present? && !@domain_block.stricter_than?(existing_domain_block)
-        @domain_block.save
+        @domain_block.validate
```

Comment:
There was no bug here since the condition implied that `@domain_block` would not be valid. But rather than relying on this brittle logic, call `#validate` rather than `#save` since that's what we really want.

Reply:
Could you remove this line entirely? It looks like the reason for the `save` or `validate` is to populate `errors`, which would be useful for either a) there is a downgrade being stopped and you want to show the flash message about downgrade but also the other errors, b) further down in the regular `save` check if we have other errors unrelated to downgrade.

The first case maybe isnt important and that path would be fine with just the flash message but not having errors populated ... the second case would be handled separately b/c there's another `save` call (L51) there.

Bigger change than what you have here would be to pull out the entire "is this a downgrade attempt?" into a `before_action` and not even need to interact with the i-var at all.

All that said -- if you want to keep this a small change, I agree with what you are observing that the `save` being called here was never going to succeed and thus the `validate` only is fine.